browser/help/topics/v4-hts.md

```diff
@@ -66,11 +66,336 @@ The above example will retrieve the entire frequency struct for each variant. To
+  - `pangolin_largest_ds`: Largest delta [score](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02664-4) across 2 splicing consequences, which reflects the probability of the variant being splice-altering. If a variant is predicted to fall within multiple genes, score is across all relevant genes. Note that there is different behavior from spliceAI: splice gain is noted as the increase in splice usage (scores range from 0 to 1) and splice loss as the decrease in splice usage (scores range from 0 to -1). The Pangolin authors proposed a cutoff of >|0.2| for the predicted increase or decrease in splice site usage.
+  - `phylop`: Base-wise conservation [score](https://pubmed.ncbi.nlm.nih.gov/37104612/) across the 241 placental mammals in the [Zoonomia](https://zoonomiaproject.org/) project. Score ranges from -20 to 9.28, and reflects acceleration (faster evolution than expected under neutral drift, assigned negative scores) as well as conservation (slower than expected evolution, assigned positive scores).
+  - `sift_max`: [Score](https://www.nature.com/articles/nprot.2009.86) reflecting the scaled probability of the amino acid substitution being tolerated, ranging from 0 to 1. Scores below 0.05 are predicted to impact protein function. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript.
+  - `polyphen_max`: [Score](https://www.nature.com/articles/nmeth0410-248) that predicts the possible impact of an amino acid substitution on the structure and function of a human protein, ranging from 0.0 (tolerated) to 1.0 (deleterious). We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript.
+
+#### gnomAD v4.1 genomes Hail Table annotations
+
+The v4 genomes Hail Table annotation schema is the same as the exomes schema, with only a few minor differences:
+
+Global fields
+
+- `interval_qc_parameters`: Only the v4 exomes HT has this global annotation
+
+Row fields
+
+- `fafmax`: This annotation is stratified by subset in the v4 exomes, but the v4 genomes annotation does not have any subset stratification
+- `region_flags`: The v4 exomes Hail Table has the following fields that are not present in the struct on the v4 genomes Hail Table:
+  - `fail_interval_qc`
+  - `outside_ukb_capture_region`
+  - `outside_broad_capture_region`
+- `allele_info`: The v4 exomes Hail Table has an additional field in this struct, 'has_star', that is not present in the struct on the v4 genomes Hail Table
+- `info`: Sibling singletons were used to train the variant QC models for the v4 exomes but were not used in the v4 genomes variant QC
+
+#### gnomAD v4.1 joint frequency Hail Table annotations
+
+The v4 genomes Hail Table annotation schema is the same as the exomes schema, with only a few minor differences:
+
+Global fields
```

Comment:
The Global fields have changed as in: : gs://gnomad-tmp/release/4.1/ht/joint/gnomad.joint.v4.1.faf.filtered.ht
```
Global fields:
    'versions': struct {
        exomes: str, 
        genomes: str
    } 
    'genomes_globals': struct {
        freq_meta: array<dict<str, str>>, 
        freq_meta_sample_count: array<int32>, 
        faf_meta: array<dict<str, str>>, 
        age_distribution: struct {
            bin_edges: array<float64>, 
            bin_freq: array<int32>, 
            n_smaller: int32, 
            n_larger: int32
        }
    } 
    'exomes_globals': struct {
        freq_meta: array<dict<str, str>>, 
        freq_meta_sample_count: array<int32>, 
        faf_meta: array<dict<str, str>>, 
        age_distribution: struct {
            bin_edges: array<float64>, 
            bin_freq: array<int32>, 
            n_smaller: int32, 
            n_larger: int32
        }
    } 
    'joint_globals': struct {
        freq_meta: array<dict<str, str>>, 
        freq_index_dict: dict<str, int32>, 
        faf_meta: array<dict<str, str>>, 
        faf_index_dict: dict<str, int32>, 
        freq_meta_sample_count: array<int32>, 
        age_distribution: struct {
            bin_edges: array<float64>, 
            bin_freq: array<int32>, 
            n_smaller: int32, 
            n_larger: int32
        }
    } 
```